### PR TITLE
[SITES-479 | SITES-447] Feature/shiny header layouts

### PR DESCRIPTION
**Pull request by ltankey**
Notes

There are some text-size issues that will be resolved by #292 

This implements the ui-kit 1.5. This also implements the new popular now links hardcoded to the top of the template.

image

image

image

### CODE REVIEW

File: app/views/layouts/root.html.haml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        %h1
+          %span.govau--text
+          %span.visuallyhidden
+            GOV.AU homepage
+        %p
+          Your guide to finding and using Australian Government services.
+      .links-group
+        %h2 Popular links
+        %ul
+          %li
```
Comment (maxious) [inline app/views/layouts/root.html.haml]: As per SITES-447 these should be:

Public Holiday dates - /times-and-dates/australian-public-holidays
Lodge your tax return
Working in Australia
Visiting Australia
What can you bring to Australia
Citizenship
School Holiday Dates - /times-and-dates/school-holidays-and-term-dates
Government Jobs

File: lib/tasks/ui-kit.rake [hunk 2]
```diff
@@ -16,7 +16,9 @@ namespace :ui_kit do
       file_name = "vendor/assets/"+destination
       text = File.read(file_name)
       text = text.gsub("asset-data-url('build/latest/img/icons/'", "asset-data-url('icons/'")
+      text = text.gsub("url('../latest/img/icons/", "asset-url('icons/")
```
Comment (maxious) [inline lib/tasks/ui-kit.rake]: This regex is redundant but more descriptive than the one added further down.

**Comment by maxious** [comment]
LGTM

**Comment by micapam** [comment]
LGTM

File: app/views/application/_feedback_and_menu.html.haml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    %li
+      = link_to 'Ministers', '#'
+    %li
+      = link_to 'News', '#'
+    %li
+      = link_to 'Departments & Agencies', '#'
+    %li
+      = link_to 'Contact us', '#'
+  .feedback
+    =link_to 'Give feedback', '#', class: 'button--feedback'
```
Comment (tessereth) [inline app/views/application/_feedback_and_menu.html.haml]: You may wish to add a note that this file is currently unused

File: app/views/layouts/root.html.haml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            = link_to 'School Holiday dates', '/times-and-dates/school-holidays-and-term-dates'
+          %li
+            = link_to 'Government Jobs', '#'
+
+
+  = content_for :meta_description do
+    - if node.short_summary
+      = content_for :meta_description do
+        = node.short_summary
```
Comment (tessereth) [inline app/views/layouts/root.html.haml]: Why are there two nested `content_for :meta_description` calls here?

File: app/views/layouts/section_home.html.haml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        %h1
+          = @section.name
+        %p
+          = @section.summary
+
+
+  = content_for :meta_description do
+    - if node.short_summary
+      = content_for :meta_description do
+        = node.short_summary
```
Comment (tessereth) [inline app/views/layouts/section_home.html.haml]: similarly the meta_description is repeated here

**Comment by tessereth** [comment]
LGTM pending fixing the things I mentioned in other comments

**Pull request merged**